packages/repco-prisma/prisma/schema.prisma

```diff
@@ -377,6 +377,18 @@ model Transcript {
   MediaAsset MediaAsset @relation(fields: [mediaAssetUid], references: [uid])
 }
 
+/// @repco(Entity)
+model Translation {
```

Comment:
Let's leave this out for now, it's not yet used in core and I think we should talk about internationalization a bit more before.